README.md

```diff
@@ -6,14 +6,25 @@ In order to use this plugin, Multus must be installed on all hosts and `NetworkA
 
 ## Overview
 
-First create network attachment definition. This object specifies to which Open vSwitch bridge should the pod be attached and what VLAN ID should be set on the port. For more information, check [plugin documentation](docs/cni-plugin.md).
+TODO: nice doc for scheduling
+
+TODO first setup your bridges on nodes
+
+OVS CNI plugin includes scheduling support, pod will be scheduled only on nodes that have requested bridges available. In order to enable this support, any OVS bridge available for pods must be marked with `external_ids:ovs-cni-resource=<name-of-your-resource>`. For more information, check [plugin documentation](docs/cni-plugin.md).
+
```

Comment:
why do we require this additional step by our users? why isn't any bridge free game for scheduling?

Reply:
A) It allows you to use different bridge names for the same network connectivity (not really necessary since you can name your bridge however you want, but it matches OVN logic with ovn-bridge-mappings).
B) You have 10 nodes, on 5 of them, you configure your br5 and you expose it to OVS-CNI. Someone creates a bridge named br5 on different node, without you knowing, providing different connectivity.
C) All OVS bridges would be listed as Node resources, we don't want to expose OVN bridge (now), openshift-sdn bridge and others.

I can change it to expose all bridges, but I think it is worth requiring one extra step to filter out unrelated bridges. Your call.

Reply:
Oh, I see. It makes sense to allow modifying the exported resource name, and to disable it completely. But let us flip the default: if the bridge has it set to null, do not expose the resource. If it is a string, expose it as "string". If it is missing, use the bridge own name.

This way we keep our flexibility, but allow for almost-zero config.

Reply:
Good, in this PR I just expose all bridges as resources, will add the filter/rename functionality later.